addons/rook/template/base/install.sh

```diff
@@ -117,6 +117,10 @@ function rook() {
 
     # migrate from Longhorn storage if applicable
     rook_maybe_migrate_from_longhorn
+
+    # wait for all pods in the rook-ceph namespace to rollout
+    log "Awaiting Rook rollout in rook-ceph namespace"
+    rook_maybe_wait_for_rollout
```

Comment:
should we add this to the rook_already_applied function as well?

Reply:
Added: https://github.com/replicatedhq/kURL/pull/4101/files#diff-5041c635371838ae5901774a8bcf202a1543309d1462226eb36278b4a07da8f9R134